pkg/binding/event_message.go

```diff
@@ -63,7 +63,7 @@ func (m *EventMessage) SetEvent(e event.Event) error {
 
 var _ Message = (*EventMessage)(nil) // Test it conforms to the interface
 
-func EventContextToBinaryEncoder(c event.EventContext, b BinaryEncoder) (err error) {
+func eventContextToBinaryEncoder(c event.EventContext, b BinaryEncoder) (err error) {
```

Comment:
what does this method do?

Reply:
It's an helper to set attributes/extensions of the event context to the `BinaryEncoder`